Formula/pdmenu.rb

```diff
@@ -7,24 +7,19 @@ class Pdmenu < Formula
 
   head "git://git.joeyh.name/pdmenu"
 
-  depends_on "gettext" => :build
-  depends_on "s-lang" => :build
+  depends_on "coreutils"
```

Comment:
Note that it built fine without `coreutils` previously, so I really doubt this is needed.

Reply:
`coreutils` was already installed in my Mac. I assure you that it does not work without. It's not difficult to test. 

It worked because I called `ginstall` instead of `install`, and didn't know that I have to add this dependency.

Reply:
I know it's not difficult to test. It was tested in CI previously, where `coreutils` was not installed.